script/postgres-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+docker run --rm \
+           -e POSTGRES_USER=vega \
+           -e POSTGRES_PASSWORD=vega \
+           -e POSTGRES_DB=vega \
+           -p 5432:5432 \
+           timescale/timescaledb:2.6.0-pg14
```

Comment:
Is that the last version?

Reply:
No, that needs to be updated, I copied it from the old Data-Node project, will update